README.rst

```diff
@@ -26,11 +26,19 @@
 SKOPS
 =====
 
-This library helps you share your scikit-learn based models and put them in
-production.
+``skops`` is a Python library helping you share your `scikit-learn
+<https://scikit-learn.org/stable/>`__ based models and put them in production.
+At the moment, it includes the following features:
+
+- ``skops.hub_utils``: tools to create a model repository to be stored on
+  `Hugging Face Hub <https://hf.co/models>`__, mainly through
+  ``skops.hub_utils.init`` and ``skops.hub_utils.push``.
+- ``skops.card``: tools to create a model card explaining what the model does
```

Comment:
How about adding something along the lines of "automatically populating" the model card? I know not everything is populated automatically, but some stuff is, which saves the user time.